migrations/src/main/resources/application.properties.template

```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 #
 
-za.co.absa.enceladus.menas.mongo.connection.string=mongodb://localhost:27017
-za.co.absa.enceladus.menas.mongo.connection.database=menas
```

Comment:
Change it in tests as well?

Reply:
What do you mean? It's changed in the tests as well already as far as I can see.